test.js

```diff
@@ -3,8 +3,8 @@ const toMultiaddr = require('./')
 
 test('should convert URIs to multiaddrs', (t) => {
   const data = [
-    ['/ip4/127.0.0.1/http', 'http://127.0.0.1'],
-    ['/ip6/fc00::/http', 'http://[fc00::]'],
+    ['/ip4/127.0.0.1/tcp/80/http', 'http://127.0.0.1'],
```

Comment:
Perhaps we should keep tests for both versions?

Reply:
You mean `ip4` and `ip6`?  Both tests have been kept, I just updated to include the port. Pesky diff.